src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -249,6 +318,9 @@ void NSModel<SortPolicy>::Serialize(Archive& ar,
                                     const unsigned int /* version */)
 {
   ar & data::CreateNVP(treeType, "treeType");
+  ar & data::CreateNVP(leafSize, "leafSize");
+  ar & data::CreateNVP(tau, "tau");
+  ar & data::CreateNVP(rho, "rho");
```

Comment:
I agree. Done in: https://github.com/mlpack/mlpack/pull/747/commits/2698225eabde8bf2022d62209b965509d3207f73